backend/server/routers/courses.py

```diff
@@ -178,3 +178,25 @@ def unselectCourse(userData: UserData, lockedCourses: list, unselectedCourse: st
     affectedCourses = User(fixUserData(userData.dict())).unselect_course(unselectedCourse, lockedCourses)
 
     return {'affected_courses': affectedCourses}
+
+@router.post("/coursesUnlockedWhenTaken/{courseToBeTaken}")
```

Comment:
what happened to the fastAPI stuff? stuff like the responseModel (a list of strings), and example output are very useful for the frontend team to understand how to use this API.

Reply:
Yeah with that the route was returning just {"courses_state": {}} but I'll try and add it back in

Reply:
Added